sdk/go/pulumi/context.go

```diff
@@ -1476,6 +1483,9 @@ func (ctx *Context) RegisterPackageRemoteComponentResource(
 func (ctx *Context) RegisterPackage(
 	in *pulumirpc.RegisterPackageRequest,
 ) (*pulumirpc.RegisterPackageResponse, error) {
+	if !ctx.state.supportsPackageReferences {
+		return nil, errors.New("the Pulumi CLI does not support package references. Please update the Pulumi CLI")
```

Comment:
Again might be worth updating this to mention parameterization too?

Reply:
Yeh this is user facing, parameterization is probably the more important thing. Maybe we should just call the feature flag "parameterization" as well 🤔